src/main/java/com/akademia/educators_back/controller/CommentController.java

```diff
@@ -0,0 +1,38 @@
+package com.akademia.educators_back.controller;
+
+import com.akademia.educators_back.service.CommentServiceImpl;
+import com.akademia.educators_back.service.impl.CommentService;
+import com.akademia.educators_back.to.CommentTo;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/comment")
+public class CommentController {
+    private CommentService commentService;
+
+    public CommentController(CommentServiceImpl commentService) {
+        this.commentService = commentService;
+    }
+
+    @GetMapping("/api/commentList")
```

Comment:
w takim wypadku ścieżka będzie wyglądać: /api/comment/api/commentlist  czyli coś za dużo.  Wg zadanka to: adnotacja nad główną klasą podaje nam główny zasób "/api/comment" a każdy kontroler wchodzi dalej po tej ścieżce. W tym wypadku powinno być "/list" nie trzeba dodawać "comment" bo ścieżka już wskazuje w jakim zasobie jesteśmy: "/api/comment/list"